cli/loader/src/lib.rs

```diff
@@ -521,11 +521,15 @@ impl Loader {
             .and_then(|n| n.to_str())
             .and_then(|file_name| self.language_configuration_ids_by_file_type.get(file_name))
             .or_else(|| {
-                path.extension()
-                    .and_then(|extension| extension.to_str())
-                    .and_then(|extension| {
-                        self.language_configuration_ids_by_file_type.get(extension)
-                    })
+                let mut path = path.to_owned();
+                let mut extensions = std::collections::VecDeque::new();
```

Comment:
Wasn't sure if this was worth a `use` at the top of the file since it's only being used here.